### PR TITLE
B-20528-INT Return assignee office users in queue payload

### PR DESCRIPTION
**Pull request by paulstonebraker**
## Summary

This work adds data to the queue payloads, for assigning office users to moves in future work. It takes the GBLOC of the currently logged in user, and the role corresponding to the queue, and returns users of the same role within the GBLOC.

### How to test

1. Login to the office app as a Services Counselor
2. Open the network tab and find the response data
3. Run the following SQL on your local db
```
select ou.id, ou.first_name, ou.last_name, to2.gbloc, r.role_name, p.privilege_name from office_users ou 
join users u on u.id = ou.user_id 
join users_roles ur on ur.user_id = u.id
join roles r on r.id = ur.role_id 
join transportation_offices to2 on ou.transportation_office_id = to2.id
left join users_privileges up on u.id = up.user_id 
left join "privileges" p on p.id = up.privilege_id 
where r.role_type = 'services_counselor'
and to2.gbloc = 'KKFA'
and u.active = true
```
inserting your gbloc into the final line
4. Compare the data and make sure they are the same
5. 
image
Screenshot 2024-07-22 at 3 43 59 PM
6. Do the same for TOO and TIO
7. Try and break it

### CODE REVIEW

File: swagger-def/ghc.yaml [hunk 1]
```diff
@@ -6195,6 +6195,26 @@ definitions:
...
         type: string
     type: object
+  AvailableOfficeUsers:
+    type: array
+    items:
+      $ref: '#/definitions/AvailableOfficeUser'
+  AvailableOfficeUser:
+    type: object
+    properties:
+      id:
```
Comment (pambecker) [inline swagger-def/ghc.yaml:6205]: What does this id represent?  is it different than the userId vaue?

Comment (paulstonebraker) [reply swagger-def/ghc.yaml:6205]: Its the office user ID, should I update the name to represent that?

**Review by pambecker — APPROVED** [review]
Looks good!  I tested multiple scenarios, an office user with single and multiple roles, with and w/out safety privileges, active and non active, all work as expected.

**Comment by loganwc** [comment]
Test passes on my local

**Pull request merged**